apps/api/src/evm/protocols/governorBravo/utils.ts

```diff
@@ -0,0 +1,27 @@
+const UNTITLED_PROPOSAL = 'Untitled Proposal';
```

Comment:
If the proposal doesn't have a title, we shouldn't store the title on the API side, we should leave it empty so the UI can decide what to show. Having it on the API makes it more hard to change and for integrators to detect. In our UI if there is no title we could show `Proposal #12` instead, in italic.